frontend/src/lib/utils/product-intents.ts

```diff
@@ -17,13 +17,29 @@ export enum ProductIntentContext {
     // Feature Flags
     FEATURE_FLAG_CREATED = 'feature flag created',
 
+    // Error Tracking
+    ERROR_TRACKING_EXCEPTION_AUTOCAPTURE_ENABLED = 'error_tracking_exception_autocapture_enabled',
+    ERROR_TRACKING_USER_GROUP_CREATED = 'error_tracking_user_group_created',
+    ERROR_TRACKING_ALERT_CREATED = 'error_tracking_alert_created',
+
+    // TODO: can only be tracked server side. Is there a helper or just copy logic from api/team.py
+    // ERROR_TRACKING_UPLOADED_SOURCE_MAP = 'error_tracking_uploaded_source_map',
+
+    // TODO: should these be intents or are they just regular product usage?
+    // ERROR_TRACKING_FILTERED_ISSUES = 'error_tracking_filtered_issues',
+    // ERROR_TRACKING_ISSUE_ASSIGN = 'error_tracking_issue_assign',
+    // ERROR_TRACKING_ISSUE_SET_STATUS = 'error_tracking_issue_set_status',
+    // ERROR_TRACKING_VIEWED_ISSUE = 'error_tracking_viewed_issue',
```

Comment:
Left some open questions here for a couple of places I wanted to add more tracking but wasn't so sure

Reply:
So the purpose of intents (the _intent_ of them, if you will 😅 🥁 ) is to capture the first moment that we're pretty certain that someone is interested in using the product, not just clicking around. So I think clicking either of the buttons here is a good start:

<img width="1063" alt="image" src="https://github.com/user-attachments/assets/b022c85f-70b0-47b3-aa7e-0ab923bffd78" />

The others are deeper - like you literally can't get to the others without doing these first two, so you don't need them (the deeper ones) because at that point the intent has already been registered.

The deeper ones should definitely be tracked but probs just with standard events, not product intents.